### PR TITLE
Filtering by min average ratings

### PR DESCRIPTION
**Pull request by krisx303**

### CODE REVIEW

**Comment by krisx303** [comment]
Without filtering:
image
With filtering by min rating 3.0:
image

All empty stars means that there is no ratings yet

File: src/transport/HttpClient.ts [hunk 1]
```diff
@@ -22,6 +22,7 @@ type BoardNotePageContent = { elements: ElementDto[]; paths: PathDto[] };
 type BoardNotePageContentWithVersion = { content: BoardNotePageContent; version: number };
 type DocumentNotePageContentWithVersion = { content: string; version: number };
 export type PermissionDto = { user: User; access: string }
+export type RatingStats = { average: number; count: number; };
```
Comment (krisx303) [inline src/transport/HttpClient.ts:25]: Do we want to display number of ratings in this view?

Comment (Adam3004) [reply src/transport/HttpClient.ts:25]: imo we do not need it, but choice is on you

Comment (Adam3004) [reply src/transport/HttpClient.ts:25]: whatever tbh

**Review by Adam3004 — APPROVED** [review]

**Comment by Adam3004** [comment]
screenshots look realy cool

**Pull request merged**